### DISCUSSION TITLE
The SessionLayer always overwrite my customize cookie

### DISCUSSION THREAD

**Discussion by evanzp0** [tokio-rs/axum#1474] [Q&A]
### Version
axum  0.5.15

### Platform
ubuntu 20 & win 11

### Description

I try to set cookie use blow code:

```rust
(AppendHeaders([(SET_COOKIE, "foo=bar"), (SET_COOKIE, "baz=qux")]),Html(""))
```
but the cookies are not found in reponse header.
I found the reason is SessionLayer gens "Set-Cookie" always overwrite my customize cookie.
It need to modify the code below 
``` rust
                        response.headers_mut().insert( // insert will overwrite other same name cookie
                            SET_COOKIE,
                             HeaderValue::from_str(&cookie.to_string()).unwrap(),
                        );
```
to this
```rust
                        response.headers_mut().append(
                            SET_COOKIE,
                            HeaderValue::from_str(&cookie.to_string()).unwrap(),
                        );
```

**Comment by davidpdrsn** [comment]
What do you mean? axum doesn’t have a `SessionLayer` afaik.

**Reply by ttys3** [reply]
maybe @evanzp0  said is this ? https://docs.rs/axum-sessions/latest/axum_sessions/

**Reply by davidpdrsn** [reply]
If so then please file an issue there. That library isn't part of this repo.
